src/currency.cpp

```diff
@@ -26,45 +26,46 @@
+	{ 3873, "", 2002,      "",                  NBSP "L.",        1, STR_GAME_OPTIONS_CURRENCY_ITL    }, ///< italian lira
+	{    4, "", 2002,      "NLG" NBSP,          "",               0, STR_GAME_OPTIONS_CURRENCY_NLG    }, ///< dutch gulden
+	{   12, "", CF_NOEURO, "",                  NBSP "Kr",        1, STR_GAME_OPTIONS_CURRENCY_NOK    }, ///< norwegian krone
+	{    6, "", CF_NOEURO, "",                  NBSP "z\xC5\x82", 1, STR_GAME_OPTIONS_CURRENCY_PLN    }, ///< polish zloty
+	{    5, "", CF_NOEURO, "",                  NBSP "Lei",       1, STR_GAME_OPTIONS_CURRENCY_RON    }, ///< romanian leu
+	{   50, "", CF_NOEURO, "",                  NBSP "p",         1, STR_GAME_OPTIONS_CURRENCY_RUR    }, ///< russian rouble
+	{  479, "", 2007,      "",                  NBSP "SIT",       1, STR_GAME_OPTIONS_CURRENCY_SIT    }, ///< slovenian tolar
+	{   13, "", CF_NOEURO, "",                  NBSP "Kr",        1, STR_GAME_OPTIONS_CURRENCY_SEK    }, ///< swedish krona
+	{    3, "", CF_NOEURO, "",                  NBSP "TL",        1, STR_GAME_OPTIONS_CURRENCY_TRY    }, ///< turkish lira
+	{   60, "", 2009,      "",                  NBSP "Sk",        1, STR_GAME_OPTIONS_CURRENCY_SKK    }, ///< slovak koruna
+	{    4, "", CF_NOEURO, "R$" NBSP,           "",               0, STR_GAME_OPTIONS_CURRENCY_BRL    }, ///< brazil real
+	{   31, "", 2011,      "",                  NBSP "EEK",       1, STR_GAME_OPTIONS_CURRENCY_EEK    }, ///< estonian krooni
+	{    4, "", 2015,      "",                  NBSP "Lt",        1, STR_GAME_OPTIONS_CURRENCY_LTL    }, ///< lithuanian litas
+	{ 1850, "", CF_NOEURO, "\xE2\x82\xA9",      "",               0, STR_GAME_OPTIONS_CURRENCY_KRW    }, ///< south korean won
+	{   13, "", CF_NOEURO, "R" NBSP,            "",               0, STR_GAME_OPTIONS_CURRENCY_ZAR    }, ///< south african rand
+	{    1, "", CF_NOEURO, "",                  "",               2, STR_GAME_OPTIONS_CURRENCY_CUSTOM }, ///< custom currency (add further languages below)
+	{    3, "", CF_NOEURO, "",                  NBSP "GEL",       1, STR_GAME_OPTIONS_CURRENCY_GEL    }, ///< Georgian Lari
+	{ 4901, "", CF_NOEURO, "",                  NBSP "Rls",       1, STR_GAME_OPTIONS_CURRENCY_IRR    }, ///< Iranian Rial
+	{   80, "", CF_NOEURO, "",                  NBSP "rub",       1, STR_GAME_OPTIONS_CURRENCY_RUB    }, ///< New Russian Ruble
+	{   24, "", CF_NOEURO, "$",                 "",               0, STR_GAME_OPTIONS_CURRENCY_MXN    }, ///< Mexican peso
+	{   40, "", CF_NOEURO, "NTD" NBSP,          "",               0, STR_GAME_OPTIONS_CURRENCY_NTD    }, ///< new taiwan dollar
+	{    8, "", CF_NOEURO, "\xC2\xA5",          "",               0, STR_GAME_OPTIONS_CURRENCY_CNY    }, ///< chinese renminbi
+	{   10, "", CF_NOEURO, "HKD" NBSP,          "",               0, STR_GAME_OPTIONS_CURRENCY_HKD    }, ///< hong kong dollar
+	{   90, "", CF_NOEURO, "\xE2\x82\xB9" NBSP, "",               0, STR_GAME_OPTIONS_CURRENCY_INR    }, ///< Indian Rupee
```

Comment:
The space doesn't seem right? All the examples I can find don't have it.

Reply:
I added extra spaces as `"\xE2\x82\xB9" NBSP` was taking quite a lot of space. So padded the rest with spaces to keep code neat.

Reply:
I mean, the `NBSP` adds a (non-breaking) space between the symbol and number in the displayed text.

Reply:
Oh yes we don't add a space after ₹ I missed it. Nice catch!